templates/modelNestedModels.hbs

```diff
@@ -5,6 +5,17 @@
  * @namespace {{className name}}
  */
 export namespace {{className name}} {
+{{#if (isMultipartSchema .)}}
```

Comment:
I wonder if we should move this into `modelGeneric.hbs` where we're doing the other multipart thing. We can declare the same namespace multiple times so it won't be problematic (I don't think). It feels a bit hidden to me to do it here.